x-pack/plugins/maps/public/routing/routes/map_app/map_app.tsx

```diff
@@ -86,7 +87,6 @@ interface Props {
   refreshConfig: MapRefreshConfig;
   setRefreshConfig: (refreshConfig: MapRefreshConfig) => void;
   isSaveDisabled: boolean;
-  clearUi: () => void;
```

Comment:
clearUi no longer needed since each savedMap has its own reduce store. Since a store is not being shared, there is nothing to clean up